vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +96,126 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
+                localizedText.setText(title.getText());            
+            } else {
+                
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(title.getExhibitionLanguageId()).orElse(null);
+                if(exhibitionLanguage != null) {
+                    localizedText = new LocalizedText(exhibitionLanguage, title.getText());
+                    exhibitionAboutPage.getExhibitionTitles().add(localizedText);
+                    exhibitionLanguage.getLocalizedTexts().add(localizedText);
+                }
+            }
+            
+        }
+
+    }
+    
+    /**
+     * This method maps the description in ExhibitionAboutPage, and add that to
+     * exhibitionTextDescriptions list for each user selected Exhibition Language.
+    */
+    public void setAboutPageDescription(LocalizedTextForm aboutPageText, ExhibitionAboutPage exhibitionAboutPage) {
+        if(aboutPageText!=null) {
+            LocalizedText localizedText = localizedTextRepo.findById(aboutPageText.getLocalisedTextId()).orElse(null);
+            
+            if(localizedText != null) {
+                localizedText.setText(aboutPageText.getText());
+            } else {
+                ExhibitionLanguage exhibitionLanguage = exhibitionLanguageRepository.findById(aboutPageText.getExhibitionLanguageId()).orElse(null);
+                if(exhibitionLanguage != null) {
+                    exhibitionAboutPage.getExhibitionTextDescriptions().add(new LocalizedText(exhibitionLanguage, aboutPageText.getText()));
+                }
+            }        
+        }
+    }
+       
+    /**
+     * Creates About Page form object
+     */
+    @Override
+    public AboutPageForm createAboutPageForm() {
+        ExhibitionAboutPage exhibitionAboutPage = getExhibitionAboutPage(); 
+
+        AboutPageForm aboutPageForm=new AboutPageForm();
+        aboutPageForm.setAboutPageText(exhibitionAboutPage.getAboutPageText());
+        aboutPageForm.setTitle(exhibitionAboutPage.getTitle());        
+        
+        IExhibition startExhibtion = exhibitionManager.getStartExhibition();
+        
+        
+        
+        IExhibitionLanguage defaultLanguage = exhibitionManager.getDefaultLanguage(startExhibtion);
+                
+               
+        
+        aboutPageForm.getTitles().add(createLocalizedTitleForm(exhibitionAboutPage, defaultLanguage));
+        
+        aboutPageForm.getAboutPageTexts().add(createLocalizedAboutTextForm(exhibitionAboutPage, defaultLanguage));
+        
+        startExhibtion.getLanguages().forEach(language -> {
+            if(!language.isDefault()) {
+                aboutPageForm.getTitles().add(createLocalizedTitleForm(exhibitionAboutPage, language));
+                
+                aboutPageForm.getAboutPageTexts().add(createLocalizedAboutTextForm(exhibitionAboutPage, language)); 
+            }
+
+
+        });
+        return aboutPageForm;
```

Comment:
why are there so many blank lines? blank lines are another tool to organize your code and should be used strategically.